languages/rust/exercises/concept/vec-macro/.meta/example.rs

```diff
@@ -0,0 +1,39 @@
+/// vec-macro for creating empty vectors, and vectors of particular size, with vec!
+
+/// Create an empty vector
+fn create_empty() -> Vec<u8> {
+    vec![]
+}
+
+/// Create a buffer of zeroes with `count` bytes capacity.
+/// Applications use these for fill and modifying data
```

Comment:
```suggestion
/// Create a buffer of `count` zeroes.
///
/// Applications use these for filling and modifying data
```

Nits:

- we don't want them returning `Vec::with_capacity(count)`
- newline between the 'header' and content makes the rustdoc render more nicely
- grammar "fill" -> "filling"